content/docs/command-reference/data/status.md

```diff
@@ -0,0 +1,123 @@
+
+By default, `dvc data status` shows changes in the level the file/directory is
+being tracked in. With `--granular`, it will show changes in file-level.
+
+The `--unchanged` option will list all of the files that aren't listed in
+committed changes or uncommitted changes.
+
+### Output
+
+The `dvc data status` displays changes in multiple categories such as:
+
+- _Not in cache_
+- _DVC committed changes_
+- _DVC uncommitted changes_
+- _Untracked files_
+- _DVC unchanged files_
+
+The categories will only be displayed when there are changes related to those
+categories.
+
+The _DVC committed changes_ and _DVC committed changes_ shows more detailed
+state the files are in: _added_, _modified_, and _deleted_.
+
+## Options
+
+- `--granular` - Show granular, file-level information for the changes for
+  dvc-tracked directories. By default, DVC only shows the changes at the level
+  where the data is being tracked at.
+
+- `--untracked-files` - Show files that are not being tracked by DVC and Git.
+  This option can also take values: `no` and `all`. With `--untracked-files=no`,
+  this change is not shown and is a default. `--untracked-files=all` or just
+  `--untracked-files` will show untracked files.
+
+- `--unchanged` - Show unchanged DVC-tracked files.
+
+- `--json` - prints the command's output in easily parsable JSON format, instead
+  of a human-readable output.
+
+- `-h`, `--help` - prints the usage/help message, and exit.
+
+- `-q`, `--quiet` - do not write anything to standard output.
+
+- `-v`, `--verbose` - displays detailed tracing information.
+
+## Examples
+
+```dvc
+$ dvc data status
+Not in cache:
+  (use "dvc pull <file>..." to update your local storage)
+        data/data.xml
+
+DVC committed changes:
+  (git commit the corresponding dvc files to update the repo)
+        modified: data/features/
+
+DVC uncommitted changes:
+  (use "dvc commit <file>..." to track changes)
+        deleted: model.pkl
+(there are other changes not tracked by dvc, use "git status" to see)
+```
+
+This shows that the `data/data.xml` is missing from the cache, `data/features/`
+a directory, has changes that are being tracked by DVC but is not git-committed
+yet, and a file `model.pkl` has been deleted from the workspace. The
+`data/features/` directory is modified, but there is no further details to what
+changed inside. The `--granular` flag can provide more information on that.
+
+## Example: Granular output
+
+Following on from the above example, using `--granular` will show file-level
+information for the changes:
+
+```dvc
+$ dvc data status
+Not in cache:
+  (use "dvc pull <file>..." to update your local storage)
+        data/data.xml
+
+DVC committed changes:
+  (git commit the corresponding dvc files to update the repo)
+        added: data/features/foo
+
+DVC uncommitted changes:
+  (use "dvc commit <file>..." to track changes)
+        modified: data/data.xml
+(there are other changes not tracked by dvc, use "git status" to see)
```

Comment:
Why are there different changes in the two examples? Not a big deal, but I think it will be easier if we show the same example with and without `--granular`.

Reply:
Thanks for noticing this. That’s what I intended to do, but looks like I did dvc pull in the middle (or did not notice a bug).